Atomic/syscontainers.py

```diff
@@ -1157,38 +1153,8 @@ def _do_checkout(self, repo, options):
         if options["prefix"]:
             return options["values"]
 
-        sym = os.path.join(self._get_system_checkout_path(), options["name"])
-        if os.path.exists(sym):
-            os.unlink(sym)
-        os.symlink(options["destination"], sym)
-
         try:
-            if rpm_install_content["rpm_file"]:
-                RPMHostInstall.install_rpm(rpm_install_content["rpm_file"])
-            else:
-                for installed_file in rpm_install_content["new_installed_files"]:
-                    util.write_out("Created file {}".format(installed_file))
-
-            self._systemctl_command("daemon-reload")
-            if (tmpfiles_template):
-                self._systemd_tmpfiles("--create", options["tmpfilesout"])
-
-            if options["upgrade_mode"] == SystemContainers.CHECKOUT_MODE_INSTALL:
-                self._systemctl_command("enable", options["name"])
-            elif was_service_active:
-                if options["upgrade_mode"] == SystemContainers.CHECKOUT_MODE_UPGRADE_CONTROLLED:
-                    must_rollback = False
-                    try:
-                        self._systemctl_command("start", options["name"])
-                    except subprocess.CalledProcessError:
-                        must_rollback = True
-                    if must_rollback:
-                        util.write_err("Could not restart {}.  Attempt automatic rollback".format(options["name"]))
-                        self.rollback(options["name"])
-                        self._systemctl_command("start", options["name"])
-                        return {}
-                else:
-                    self._systemctl_command("start", options["name"])
+            self.checkout_container_wrapup(rpm_install_content, tmpfiles_template, options, was_service_active)
```

Comment:
could we rename it to `finalize_checkout` or something similar?

Reply:
sounds good